test/dynamo/test_optimizers.py

```diff
@@ -44,6 +44,10 @@ def fn():
 
 
 def make_test(optim_cls, closure=None, **kwargs):
+    # Remove this conditional when #118230 is fixed
```

Comment:
@mlazos FYI in correcting the test failures, I added this since the single tensor Adamax doesn't pass the test. This is also related to #118885!